### PR TITLE
tidyr v1.0.0 udpates

### PR DESCRIPTION
**Pull request by JosiahParry**
Updated your slides to reflect changes to tidyr since v1.0.0.
Also includes arrow as package for reading data.

### CODE REVIEW

File: intro-to-tidyverse.rmd [hunk 1]
```diff
@@ -398,7 +397,7 @@ All packages share an underlying design philosophy, grammar, and data structures
 
 --
 
-* Core packages - `readr`, `tidyr`, `dplyr`, `ggplot2`
+* Core packages - `readr`, `tidyr`, `dplyr`, `ggplot2`, `purrr`, `stringr`, `forcats`
```
Comment (JosiahParry) [inline intro-to-tidyverse.rmd:400]: added additional packages to "core packages"

File: intro-to-tidyverse.rmd [hunk 2]
```diff
@@ -483,23 +483,39 @@ untidy_df %>%
...
-Read in data with `readr`, `haven`, `readxl`
+Read in data with `readr`, `arrow`, `haven`, `readxl`
 
-### `readr`
+#### `readr`
 * `read_csv()`, `read_tsv()`, `read_delim()`
 
-### `haven`
+#### `arrow`
+* `read_arrow()`, `read_feather()`, `read_parquet()`, `read_*_arrow()`
```
Comment (JosiahParry) [inline intro-to-tidyverse.rmd:513]: addition of arrow

File: intro-to-tidyverse.rmd [hunk 3]
```diff
@@ -762,7 +778,7 @@ mtcars %>%
   mutate(miles_per_liter = mpg * 3.78541, #<<
          miles_per_gallon = miles_per_liter / 3.78541) %>% #<<
   select(miles_per_liter, mpg,miles_per_gallon) %>%
-  head(10)
+  slice(1:10)
```
Comment (JosiahParry) [inline intro-to-tidyverse.rmd:781]: attempting to use `slice()` to keep it with a focus on dplyr

File: intro-to-tidyverse.rmd [hunk 4]
```diff
@@ -930,9 +946,11 @@ The goal of tidyr is to help you create tidy data. Tidy data is data where:
 --
 
 
-### Make Taller and Make Wider
-* `gather()` - takes multiple columns, and gathers them into key-value pairs: it makes “wide” data longer.
-* `spread()` - takes two columns (key & value), and spreads into multiple columns: it makes “long” data wider.
+### Make Longer and Make Wider
+* `pivot_longer()` - takes multiple columns, and makes the data "longer" by reducing the number of columns and increasing the number of rows.
```
Comment (JosiahParry) [inline intro-to-tidyverse.rmd:950]: did my best to take from `vignettes("pivot")`. The functions are much more flexible now and as such previous definitions are too restrictive.

File: intro-to-tidyverse.rmd [hunk 5]
```diff
@@ -972,16 +994,20 @@ untidy_df %>%
 
 ```{r, highlight.output = c(2, 4, 9), eval = FALSE}
 untidy_df %>% 
-  gather(gender_year, income, male_2016:female_2018) %>% 
-  separate(gender_year, into = c("gender", "year")) #<<
+  pivot_longer(cols = male_2016:female_2018, 
+               names_to = c("gender", "year"),
+               names_sep = "_", #<<
```
Comment (JosiahParry) [inline intro-to-tidyverse.rmd:999]: this is an interesting new addition that seems like it bypasses `separate()` definitely useful to show both.

File: intro-to-tidyverse.rmd [hunk 6]
```diff
@@ -1007,17 +1033,59 @@ tidy_df %>%
...
   unite("gender_year", c("gender", "year"), sep = "_") %>% 
-  spread(gender_year, income) #<<
+  pivot_wider(names_from = gender_year, values_from = income)#<<
+```
+
+---
+
+class: center, middle, inverse
+
+# SIDEBAR!
```
Comment (JosiahParry) [inline intro-to-tidyverse.rmd:1051]: Including janitor as a way to talk about the importance of consistent styling and what I would consider tidy col names

File: intro-to-tidyverse.rmd [hunk 7]
```diff
@@ -1153,8 +1223,8 @@ gapminder_df %>%
 
 ```{r,  fig.height = 4}
 gapminder_df %>%
-  filter(country == "China" | country == "United States") %>% #<<
-  ggplot(aes(x = year, y = lifeExp, colour = country)) +
+  filter(country %in% c("China", "United States")) %>% #<<
```
Comment (JosiahParry) [inline intro-to-tidyverse.rmd:1226]: used `%in%` logical throughout

**Comment by jthomasmock** [comment]
Thanks my dude!

**Pull request merged**